ttt-ttpe/src/main/java/com/skynav/ttpe/render/svg/SVGRenderProcessor.java

```diff
@@ -563,14 +579,16 @@ private Element renderBlock(Element parent, BlockArea a, Document d) {
         }
         if (Documents.getAttribute(eBlockGroup, SVGDocumentFrame.classAttrName, null) == null)
             maybeMarkClasses(eBlockGroup, a, "block");
-        return eBlockGroup;
+        if (eBlockGroup.hasChildNodes() || currentPass == Pass.MAIN)
```

Comment:
In order to reduce the amount of trivial unit test changes in this PR, I added code to prune empty element hierarchies in the new passes, with empty hierarchies still present in the main pass. For TTMLs without any backgrounds or outlines, this should result in the output being the same as before. This code could be simplified somewhat by pruning empty hierarchies in all passes or never pruning them at all, but at the expense of much more unit test churn in the PR.